workers/r_processors/cel_to_pcl.r

```diff
@@ -0,0 +1,40 @@
+ProcessCelFiles <- function(process.dir, organism.code, output.file) {
+    library("affy")
+    library("affyio")
+    ##files now holds a list of all celfiles in the directory to be processed
+    files = list.celfiles(process.dir)
```

Comment:
So here you would do

```R
files <- affy::list.celfiles(process.dir)
```

Also note the use of `<-` rather than `=` for assignment